docs/source/overview/cache_groups.rst

```diff
@@ -31,4 +49,273 @@ All servers have to have a :term:`Physical Location`, which defines their geogra
+
+When this *is* a "true" value, the closest Cache Group will be chosen if and only if any set of Fallbacks_ configured on the Cache Group has already been exhausted and no available Cache Groups were found..
+
+.. seealso:: :ref:`health-proto`
+
+.. table:: Aliases
+
+	+--------------------------+----------------------+----------------------------------------+
+	| Name                     | Use(s)               | Type(s)                                |
+	+==========================+======================+========================================+
+	| Fallback to Geo Failover | Traffic Portal forms | Unchanged (``bool``, ``Boolean`` etc.) |
+	+--------------------------+----------------------+----------------------------------------+
+
+.. _cache-group-id:
+
+ID
+--
+All Cache Groups have an integral, unique identifier that is mainly used to reference it in the :ref:`to-api`.
+
+Despite that a Cache Group's Name_ must be unique, this is the identifier most commonly used to represent a unique Cache Group in most contexts throughout :abbr:`ATC (Apache Traffic Control)`. One notable exception is in CDN :term:`Snapshots` and in routing configuration used by Traffic Router.
+
+.. _cache-group-latitude:
+
+Latitude
+--------
+The Cache Group's geomagnetic latitude for use in routing and for the purposes of `Fallback to Closest`_.
+
+.. table:: Aliases
+
+	+-----------------------+----------------------+----------------------------------------+
+	| Name                  | Use(s)               | Type(s)                                |
+	+=======================+======================+========================================+
+	| Geo Magnetic Latitude | Traffic Portal forms | Unchanged (``number``, ``float`` etc.) |
+	+-----------------------+----------------------+----------------------------------------+
+
+.. _cache-group-localization-methods:
+
+Localization Methods
+--------------------
+The :dfn:`Localization Methods` of a Cache Group define the methods by which Traffic Router is allowed to route clients to :term:`cache servers` within this Cache Group. This is a collection of the allowed methods, and the values in the collection are restricted to the following.
+
+- "Coverage Zone File" (alias ``CZ`` in source code, database entries, and :ref:`to-api` requests/responses) allows Traffic Router to direct clients to this Cache Group if they were assigned a geographic location by looking up their IP address in the :term:`Coverage Zone File`.
+- "Deep Coverage Zone File" (alias ``DEEP_CZ`` in source code, database entries, and :ref:`to-api` requests/responses) allows Traffic Router to direct clients to this Cache Group if they were assigned a geographic location by looking up their IP addresses in the :term:`Deep Coverage Zone File`.
```

Comment:
This is going to be removed from TP per #3067. want to make the necessary changes to these docs?

Reply:
I'm hesitant to do that. Because I don't want the docs for something to get into a release where it's wrong - if #3067 merges before this one I'll be happy to update it, but if this merges first #3067 should do it. That way the docs are always exactly correct for the features that exist.

Reply:
makes sense

Reply:
deep czf checkbox was removed from the cachegroup form with #4172

Reply:
But it's still present in API responses? Then this really oughtta be left in. But if it truly doesn't affect anything I need to add that as a note or something. In fact I think right now I assert that it does in fact affect things.

Reply:
yeah, you probably needs some kind of note in there because it's not as simple as setting cachegroup.localizationMethods = ['DEEP_CZ'] to enable deep routing on a cache group.